### ISSUE TITLE
Color contrast for button backgrounds

### ISSUE THREAD

**Issue by hannaeko** [zonemaster/zonemaster-gui#445]
Secondary buttons (#e8e8e8) have an insufficient contrast ratio on both primary background (#ffffff) and form background (#edebff)

**Comment by tgreenx** [comment]
Fixed by #511 in release v2025.2
